src/Core/Framework/Store/Command/StoreLoginCommand.php

```diff
@@ -74,6 +70,21 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $password = $input->getOption('password');
         $user = $input->getOption('user');
 
+        if (!$password) {
+            $passwordQuestion = new Question('Enter password');
+            $passwordQuestion->setValidator(static function ($value): string {
+                if ($value === null || trim($value) === '') {
+                    throw new \RuntimeException('The password cannot be empty');
+                }
+
+                return $value;
+            });
+            $passwordQuestion->setHidden(true);
+            $passwordQuestion->setMaxAttempts(3);
+
+            $password = $io->askQuestion($passwordQuestion);
+        }
+
```

Comment:
You don't need to check this, because the password is requierd. So please delete these lines

Reply:
Hello,

Please keep in mind that even if the option `password` has the `required` flag you can still ignore passing it as an argument and therefor it will be `null`. 

The whole purpose of this PR is to add a interactive option to supply a password for those who would like to keep sensitive passwords from leaking in the terminal history. 

Meaning your request to remove these lines invalidates this purpose.

Reply:
Hey,

that's ok. Can you add a test for the password empty case?

Reply:
Hey @shyim,

I just pushed tests for all three commands.